cmake/templates/BoostConfig.cmake.in

```diff
@@ -39,6 +39,18 @@ if(NOT TARGET "Boost::boost")
         INTERFACE_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB=1"
     )
   endif()
+
+  if(BOOST_CONFIG_LINK_OPENSSL)
+    hunter_add_package(OpenSSL)
```

Comment:
You are right I put it somewhere else, but I didn't find where. I enabled OpenSSL to Boost::boost "package" only. This is the header only part of boost. So I think install after boost is not elegant but works for me. I used the patched version of websocketpp (https://github.com/tatraian/websocketpp/tree/0.7.0-hunter).